go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/sigstore/rekor
 
-go 1.18
+go 1.19
```

Comment:
Does this bump need to happen? IIUC it means everyone that imports us must also use go 1.19.

Reply:
yes, that is true, i will drop that

Reply:
should we add a job that tests using go1.18?

Reply:
Honestly I don't think we need parallel testing here, it's somewhat overkill and just magnifies flakes.